mesonbuild/modules/i18n.py

```diff
@@ -92,7 +92,7 @@ def merge_file(self, state, args, kwargs):
         outputs = mesonlib.substitute_values([output], values)
         output = outputs[0]
 
-        ct = build.CustomTarget(output + '_' + state.subdir + '_merge', state.subdir, state.subproject, kwargs)
+        ct = build.CustomTarget(output + '_' + state.subdir.replace("/", "__") + '_merge', state.subdir, state.subproject, kwargs)
```

Comment:
I think it would be better to use single quotes because Meson uses single quotes basically.

Reply:
@kakurasan good point. I am not a native pythonist so I had to look up how to do
string replacement and I probably just cargo cult'd the double quotes in the howto.
Will switch to single quotes.